### DISCUSSION TITLE
Blank page when clicking "Variant search" in the header

### DISCUSSION THREAD

**Discussion by nvnieuwk** [broadinstitute/seqr#2871] [Q&A] [answered]
**Describe the bug**
When clicking the "Variant Search" link in the header of the variant search page of a family, it redirects me to a blank page.

**Link to page(s) where bug is occurring**
Example with the demo:
https://seqr.broadinstitute.org/variant_search/project/R0555_seqr_demo

**Scope of the bug**
I've discovered this bug in a local installation and found out it is also there on the official site for every project/family I've tried

**Comment by nvnieuwk** [comment]
Also, Seqr doesn't show me any projects on the dasboard anymore on my local installation after going to this blank page

**Comment by hanars** [comment]
Can you please include more details about the issue you are experiencing. Has this always been in issue on your local installation, or did it develop after a recent update? Are you sure you are running the latest version of the seqr code? If you refresh the page, does that resolve the issue?

**Reply by nvnieuwk** [reply]
Thanks for the quick response

We've cloned the repository on our company GitLab on July 29th, so the local installation is the newest one. I've only done some changes on fetching alignment files for IGV, but the problem was already there from the start. I checked the official Seqr site and it contains the same issue (You can replicate it by going to the the project variant search tab and clicking on the "Variant Search" link at the top of the page). 

When I refresh the page of my local installation, it isn't blank anymore, but the dashboard remains empty until I fully restart the application. Refreshing also solves the blank page on the official Seqr site, but the projects are still in the dashboard there.

Browser: Mozilla Firefox 103.0
OS: Ubuntu 20.04 LTS

**Comment by hanars** [comment]
seqr is only supported in chrome, not in firefox. Can you confirm you do experience the issue in chrome?

**Reply by nvnieuwk** [reply]
Yes, I encounter the same problem in Google Chrome version 103.0.5060.134

**Comment by hanars** [comment]
I'm sorry for the delay on this, I've been out on vacation. Can you add a screenshot for the that shows the link you are referring to where this issue occurs?

**Reply by nvnieuwk** [reply]
No problem! This is a screenshot of the blank screen with the console log. This happens right after I click on "Variant Search" in the Variant search screen.
![image](https://user-images.githubusercontent.com/101190534/185058177-c9a31f3a-d617-4055-be98-4899bd66042b.png)

**Reply by hanars** [reply]
Sorry I should have been more clear, can you send a screenshot of the page you are on when you click the link that takes you to this page? This is not actually a page you should be able to access at all, theres nothing there which s why it is blank, and I'm having trouble figuring out how you are getting there.

**Reply by nvnieuwk** [reply]
No problem, hopefully this is the correct screenshot :) 
![image](https://user-images.githubusercontent.com/101190534/185150268-3f2c54c3-8af1-42ee-b2b8-4ccd2e54b635.png)

**Reply by hanars** [reply]
So that link doesn't do anything. I'll put up a fix so it no longer looks like a link to prevent this confusion, but you should just not click there as there is no page associated with it. You are already on the variant search page.

**Reply by nvnieuwk** [reply]
Thank you! And yes I thought so, just wanted to report it :). I thought it would maybe reset the variant search page.

**Reply by hanars** [reply]
Yes, thanks for letting us know as it is confusing that there is something that looks like a link that you aren't supposed to click

**Comment by hanars** [comment, marked as answer]
I have disabled the link to prevent this confusion in the future

**Reply by nvnieuwk** [reply]
Thank you!
